src/Compilers/VisualBasic/Test/CommandLine/CommandLineTests.vb

```diff
@@ -2068,8 +2073,9 @@ End Module").Path
             Assert.Equal(LanguageVersion.VisualBasic15_3, LanguageVersion.VisualBasic15_3.MapSpecifiedToEffectiveVersion())
             Assert.Equal(LanguageVersion.VisualBasic15_5, LanguageVersion.VisualBasic15_5.MapSpecifiedToEffectiveVersion())
             Assert.Equal(LanguageVersion.VisualBasic16, LanguageVersion.VisualBasic16.MapSpecifiedToEffectiveVersion())
+            Assert.Equal(LanguageVersion.VisualBasic16_9, LanguageVersion.VisualBasic16_9.MapSpecifiedToEffectiveVersion())
             Assert.Equal(LanguageVersion.VisualBasic16, LanguageVersion.Default.MapSpecifiedToEffectiveVersion())
```

Comment:
>LanguageVersion.Default [](start = 56, length = 23)

Let's confirm what version `Default` should map to. I think we could have it point to 16.9 now.

Reply:
Prior to moving to VB16 we had mapping work this way:
```
            Select Case version
                Case LanguageVersion.Latest
                    Return LanguageVersion.VisualBasic15_5
                Case LanguageVersion.Default
                    Return LanguageVersion.VisualBasic15
                Case Else
                    Return version
            End Select
```
I am following the same pattern.

---
In reply to: [556982329](https://github.com/dotnet/roslyn/pull/50414#discussion_r556982329) [](ancestors = 556982329)

Reply:
I don't believe we can update `Default` until we have a new major VS version. @jaredpar, can you weigh in as well?

---
In reply to: [557448246](https://github.com/dotnet/roslyn/pull/50414#discussion_r557448246) [](ancestors = 557448246,556982329)